### PR TITLE
[REGEDIT] Use InitCommonControlsEx instead of InitCommonControls

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: N/A

## Proposed changes

- Call `comctl32!InitCommonControlsEx` function rather than `comctl32!InitCommonControls` funciton.

no-change
No change.

### CODE REVIEW

**Comment by katahiromz** [comment]
The following images are evidence:
WinXP:
APIMonitor-InitCommonControlsEx-regedit-WinXP
Win2k3:
APIMonitor-InitCommonControlsEx-regedit-Win2k3
Win10:
APIMonitor-InitCommonControlsEx-regedit-Win10

File: base/applications/regedit/main.c [hunk 1]
```diff
@@ -97,8 +98,9 @@ BOOL InitInstance(HINSTANCE hInstance, int nCmdShow)
     hPopupMenus = LoadMenuW(hInstance, MAKEINTRESOURCEW(IDR_POPUP_MENUS));
 
     /* Initialize the Windows Common Controls DLL */
-    // TODO: Replace this call by InitCommonControlsEx(_something_)
-    InitCommonControls();
+    icce.dwSize = sizeof(icce);
+    icce.dwICC = 0xFFFF; /* Windows does this! */
```
Comment (GeoB99) [inline base/applications/regedit/main.c:102]: Shouldn't that be `ICC_WIN95_CLASSES`?

Comment (katahiromz) [reply base/applications/regedit/main.c:102]: Windows uses this value (`0xFFFF`) as above images.

**Comment by Extravert-ir** [comment]
What's the point in keeping it like Windows (tm) here? Does it have any noticeable impact?

File: base/applications/regedit/main.c (hunk 1 above)
Comment (GeoB99) [reply base/applications/regedit/main.c:102]: I seriously have allergies when I see magic values like these, specially with no proper description or meaning about these whatsoever. Please do some research on how is this value treated and what Windows does with it, having only a `Windows does this!` comment is pretty vague.

**Comment by katahiromz** [comment]
The other value might optimize the app startup time a bit. This PR is not important.

File: base/applications/regedit/main.c (hunk 1 above)
Comment (katahiromz) [reply base/applications/regedit/main.c:102]: This `0xFFFF` value would be the combination of:
```txt
ICC_WIN95_CLASSES 0x000000FF
ICC_DATE_CLASSES 0x00000100
ICC_USEREX_CLASSES 0x00000200
ICC_COOL_CLASSES 0x00000400
ICC_INTERNET_CLASSES 0x00000800
ICC_PAGESCROLLER_CLASS 0x00001000
ICC_NATIVEFNTCTL_CLASS 0x00002000
ICC_STANDARD_CLASSES 0x00004000
ICC_LINK_CLASS 0x00008000
```

Comment (katahiromz) [reply base/applications/regedit/main.c:102]: `ICC_WIN95_CLASSES | ICC_USEREX_CLASSES | ICC_LINK_CLASS` might be good enough. `ICC_USEREX_CLASSES` is for comboboxex controls. `ICC_LINK_CLASS` is for hyperlink controls. We can ignore `ICC_STANDARD_CLASSES` flag as next article: https://stackoverflow.com/questions/33312287/is-a-button-a-common-control

Comment (katahiromz) [reply base/applications/regedit/main.c]: Now optimized.

File: base/applications/regedit/main.c [hunk 2]
```diff
@@ -2,6 +2,7 @@
  * Regedit main function
  *
  * Copyright (C) 2002 Robert Dickenson <robd@reactos.org>
+ * Copyright (C) 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```
Comment (HBelusca) [inline base/applications/regedit/main.c:5]: Not sure adding copyrights are relevant when a one-line addition is done in a large file...

**Review by HBelusca — COMMENTED** [review]
looks fine to me.

File: base/applications/regedit/main.c (hunk 2 above)
Comment (katahiromz) [reply base/applications/regedit/main.c]: Deleted.

**Pull request merged**